### PR TITLE
Uat

### PR DESCRIPTION
**Pull request by alessio-creo**
needs #46 

Needs to be filled qith uat account information

### CODE REVIEW

File: src/init/env/uat/backend.tfvars
Comment (micdes-pagopa) [inline src/init/env/uat/backend.tfvars:1]: Already done, you can discard these changes

File: src/init/env/uat/terraform.tfvars
Comment (micdes-pagopa) [inline src/init/env/uat/terraform.tfvars:1]: Already done, you can discard these changes

File: src/main/k8s/env/uat/backend.tfvars [hunk 1]
```diff
@@ -0,0 +1,4 @@
+bucket         = "terraform-backend-774300547186"
```
Comment (micdes-pagopa) [inline src/main/k8s/env/uat/backend.tfvars:1]:
```suggestion
bucket         = "terraform-backend-010158505074"
```

File: src/main/k8s/env/uat/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+tags = {
+  CreatedBy   = "Terraform"
+  Environment = "uat"
+  Owner       = "PagoPA"
+  Source      = "https://github.com/pagopa/interop-infra-probing"
+}
+
+be_prefix                    = "interop-be-probing"
+eks_cluster_name             = "interop-probing-eks-uat"
+adot_irsa_role_arn           = "arn:aws:iam::774300547186:role/adot"
```
Comment (micdes-pagopa) [inline src/main/k8s/env/uat/terraform.tfvars:14]:
```suggestion
adot_irsa_role_arn           = "arn:aws:iam::010158505074:role/adot"
```

File: src/main/k8s/env/uat/terraform.tfvars [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+fargate_profiles_roles_names = ["SystemProfile-20230228142846109700000003", "ApplicationProfile-20230228142846109000000001", "ObservabilityProfile-20230317172642820300000001"]
```
Comment (micdes-pagopa) [inline src/main/k8s/env/uat/terraform.tfvars:15]: These will need to be set after deploying `main`

File: src/main/k8s/env/uat/terraform.tfvars [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+sso_full_admin_role_name     = "AWSReservedSSO_FullAdmin_43e33324db7f1652"
```
Comment (micdes-pagopa) [inline src/main/k8s/env/uat/terraform.tfvars:16]:
```suggestion
sso_full_admin_role_name     = "AWSReservedSSO_FullAdmin_010f7fe5de34c177"
```

File: src/main/k8s/env/uat/terraform.tfvars [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+infra_repo_role_name         = "GithubActionIACRole"
+k8s_repo_role_name           = "interop-be-probing-k8s-deploy-uat"
+adot_collector_img_tag       = "v0.30.0"
+metrics_server_img_tag       = "v0.6.3"
+
+iam_users_k8s_admin           = ["alessio.creo", "alessandro.colella", "giuseppe.porro", "giuseppe.dellorusso"]
```
Comment (micdes-pagopa) [inline src/main/k8s/env/uat/terraform.tfvars:22]:
```suggestion
```

File: src/main/env/uat/backend.tfvars [hunk 6]
```diff
@@ -0,0 +1,4 @@
+bucket         = "terraform-backend-774300547186"
```
Comment (micdes-pagopa) [inline src/main/env/uat/backend.tfvars:1]:
```suggestion
bucket         = "terraform-backend-010158505074"
```

File: src/main/env/uat/terraform.tfvars [hunk 7]
```diff
@@ -0,0 +1,60 @@
+aws_region          = "eu-central-1"
+env                 = "uat"
+app_name            = "interop-probing"
+be_prefix           = "interop-be-probing"
+sso_admin_role_name = "AWSReservedSSO_FullAdmin_43e33324db7f1652"
```
Comment (micdes-pagopa) [inline src/main/env/uat/terraform.tfvars:5]:
```suggestion
sso_admin_role_name = "AWSReservedSSO_FullAdmin_010f7fe5de34c177"
```

**Comment by alessio-creo** [comment]
Triggers must be set to always because no change in the code will cause a change for the lambda deploy zip file, which will be built without dependencies.

**Pull request closed without merging**